test/config_test.dart

```diff
@@ -27,6 +27,24 @@ class SimpleModule extends Module {
   }
 }
 
+const tempDumpDir = 'tmp_test';
```

Comment:
You should be able to `import` the `wave_dumper_test.dart` file to get access to these same functions and variables rather than copy-pasting.